src/networking.c

```diff
@@ -399,6 +399,20 @@ void addReplyErrorLength(client *c, const char *s, size_t len) {
 
 /* Do some actions after an error reply was sent (Log if needed, updates stats, etc.) */
 void afterErrorReply(client *c, const char *s, size_t len) {
+    /* Update per command error stats */
+    if (c->cmd) {
+        c->cmd->failed_calls++;
+    }
+    /* Increment the error stats */
+    char* spaceloc = strchr(s, ' ');
+    size_t padding = 0;
+    // remove spurious dash
+    if (s[0] == '-') padding++;
```

Comment:
we do not want to **optionally** remove the spurious dash, this is in fact an indication that this method (`afterErrorReply`) is called in the wrong place, and given the wrong input (incomplete error message).
if we do that, the dash will always be present (and we can remove it, and even assert that it's there).

Reply:
agree. fixed in the latest commit and squash. please see my latest comment